openedx/core/djangoapps/appsembler/sites/utils.py

```diff
@@ -401,7 +401,7 @@ def compile_sass(sass_file, custom_branding=None):
     if custom_branding:
         importers = [(0, custom_branding)]
     css_output = sass.compile(
-        filename=sass_var_file,
+        dirname=(theme.path, theme.path),
```

Comment:
This is broken now, but will be fixed with the Juniper theme so this change shouldn't go to `main`.

```suggestion
        filename=sass_var_file,
```